hack/gen-api-reference-docs.sh

```diff
@@ -175,7 +175,7 @@ main() {
 
     log "SUCCESS: Generated docs written to ${out_dir}/."
     log "Opening the ${out_dir}/ directory. You can now copy these API files"
-    log "from ${out_dir}/, into the 'docs/reference/' directory of knative/docs."
+    log "from ${out_dir}/, into the 'docs/reference/api' directory of knative/docs."
```

Comment:
added this change back into your PR (sorry didn't mention I committed this last review)